open_base:

    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 100

    left_joint_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: rim_left_joint
        pid: {p: 1.0, i: 0.0, d: 1.0}

    back_joint_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: rim_back_joint
        pid: {p: 1.0, i: 0.0, d: 1.0}

    right_joint_velocity_controller:
        type: velocity_controllers/JointVelocityController
        joint: rim_right_joint
        pid: {p: 1.0, i: 0.0, d: 1.0}
